css/ace-mantis.css

```diff
@@ -294,6 +294,35 @@ i.datetimepicker {
     color: #333333;
 }
 
+.dropdown-menu .divider {
+    margin: 2px 0;
+}
+
+.list > li {
```

Comment:
Should the styles here be specific to project list rather than a generic name "list"?  You have a id `projects-list` and class `projects-searchbox`  for this already.